ai21/models/chat/document_schema.py

```diff
@@ -0,0 +1,10 @@
+from __future__ import annotations
+from typing import Optional
+from pydantic import Field
+from ai21.models.ai21_base_model import AI21BaseModel
+
+
+class DocumentSchema(AI21BaseModel):
+    id: Optional[str] = Field(None, description="Id of the document", max_length=128)
+    content: str = Field(description="Content of the document")
+    metadata: Optional[dict[str, str]] = Field({}, description="Metadata of the document")
```

Comment:
Why isn't it NOT_GIVEN?

Reply:
What do you mean? You want our users to be exposed to this internals when inspecting this class?